Single-Sign-on/faqs.md

```diff
@@ -0,0 +1,165 @@
+---
+description: Single Sign-on FAQs
+keywords: Docker, Docker Hub, SSO FAQs, single sign-on
+title: SSO FAQs
+---
```

Comment:
Could you add the following to the metadata so the FAQ doesn't list all the questions in the right navigation (mini toc on the right)

```
toc_max: 2
```

Reply:
Updated